android/src/main/java/com/sketchView/SketchView.java

```diff
@@ -47,6 +47,10 @@ public void setToolType(int toolType) {
         }
     }
 
+    public void setPenColor(int toolColor) {
+        currentTool = new PenSketchTool(this, toolColor);
```

Comment:
No need to create another PenSketchTool. I have already implemented setting pen color in the `PenSketchTool`.